projects/storefrontapp/src/app/app.module.ts

```diff
@@ -49,8 +50,8 @@ if (!environment.production) {
       featureModules: {
         productConfiguratorRulebased: {
           module: () =>
-            import('@spartacus/product-configurator/rulebased').then(
-              (m) => m.RulebasedConfiguratorModule
+            import('@spartacus/product-configurator/rulebased/cpq').then(
+              (m) => m.RulebasedCpqConfiguratorModule
             ),
         },
         productConfiguratorTextfield: {
```

Comment:
This change should be conditional based on `environment.cpq`. For example we can have 2 configs defined in this file (for vc and for cpq) and include in `imports: [...]` only one, based on the env flag.

Not super urgent, if SPA CI will pass with it. But let's not postpone it too much.

Reply:
Did that